server/views/partials/case-view.njk

```diff
@@ -65,12 +65,11 @@
                 ]
             }) }}
 
-
             {% block caseContent %}{% endblock %}
 
         </div>
     </div>
-    <form action="{{ '/pdu/'+pduCode + '/' + crn + '/convictions/' + convictionNumber + '/case-view'}}" method="POST">
+    <form data-validate-form="case-view">
```

Comment:
Should there be a an action or will the library take care of that?

Reply:
There's no need for an action as the button is just a link so the form can't actually submit, previously the whole thing wasn't wrapped in a form either but since the validation library uses the `FormData` class there needs to be one now